app/controllers/groups_controller.rb

```diff
@@ -18,7 +18,7 @@ def edit
   end
 
   def new
-    @group = Group.new
+    @group = Group.new(type: InstanceGroup)
```

Comment:
```suggestion
    @group = InstanceGroup.new
```
Geht das nicht auch? Oder kollidiert das mit den `becomes(Group)` Anweisungen in den Views?